pymilvus/client/stub.py

```diff
@@ -134,25 +134,22 @@ def __init__(self, host=None, port=None, handler="GRPC", pool="SingletonThread",
 
     @check_connect
     def _wait_for_healthy(self, timeout=30, retry=10):
-        _timeout_on_every_retry = self._kw.get("timeout", None)
```

Comment:
Thank you for your contribution. This function was modified before you submit the PR, adding the following code:
```
_timeout_on_every_retry = self._kw.get("timeout", None)
_timeout = _timeout_on_every_retry if _timeout_on_every_retry else timeout
```
It means that if Milvus's own `_kw` member contains the `timeout`, it will overwrite the `timeout` passed in by the parameter.
Can you revise it to keep this logic?